src/components/LandingPage/LandingPage.js

```diff
@@ -14,6 +15,7 @@ const LandingPage = (props) => {
         + " resources and more."
     const placeholderText = '';
     // const max_cards = 4;
```

Comment:
Is this commented-out code something that we need to keep?  I think at some point we'll need to make it configurable.

Reply:
This has already been made configurable - I will go ahead and remove it